cmd/webhook/main.go

```diff
@@ -123,6 +126,41 @@ func NewConfigValidationController(ctx context.Context, cmw configmap.Watcher) *
 	)
 }
 
+func NewConversionController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
+	var (
+		pubsubv1alpha1_ = pubsubv1alpha1.SchemeGroupVersion.Version
+		pubsubv1beta1_  = pubsubv1beta1.SchemeGroupVersion.Version
+	)
+
+	return conversion.NewConversionController(ctx,
+		// The path on which to serve the webhook
+		"/resource-conversion",
+
+		// Specify the types of custom resource definitions that should be converted
+		map[schema.GroupKind]conversion.GroupKindConversion{
+			// pubsub
+			pubsubv1beta1.Kind("PullSubscription"): {
+				DefinitionName: pubsub.PullSubscriptionsResource.String(),
+				HubVersion:     pubsubv1alpha1_,
+				Zygotes: map[string]conversion.ConvertibleObject{
+					pubsubv1alpha1_: &pubsubv1alpha1.PullSubscription{},
+					pubsubv1beta1_:  &pubsubv1beta1.PullSubscription{},
+				},
+			},
+			pubsubv1beta1.Kind("Topic"): {
+				DefinitionName: pubsub.TopicsResource.String(),
+				HubVersion:     pubsubv1alpha1_,
+				Zygotes: map[string]conversion.ConvertibleObject{
+					pubsubv1alpha1_: &pubsubv1alpha1.Topic{},
+					pubsubv1beta1_:  &pubsubv1beta1.Topic{},
+				},
+			},
+		},
+		func(ctx context.Context) context.Context {
```

Comment:
Nit: add a comment about the purpose of this argument, as in https://github.com/knative/eventing/blob/1e68f1cfe5bb4f8c67aca7b09b36b973cde278dc/cmd/webhook/main.go#L333

Reply:
https://github.com/google/knative-gcp/pull/813